src/features/network/store.ts

```diff
@@ -1,99 +1,108 @@
 import create from "zustand"
+import { NetworkId, NetworkInfo, NetworksState } from "./types"
 import { persist } from "zustand/middleware"
 import localforage from "localforage"
 import { replacer, reviver } from "shared"
-import { NetworkId, NetworkParams, NetworksState } from "./types"
 
-const devDomains = ["localhost", "liftedinit.tech"]
-
-const initialState = {
-  activeId: 0,
-  nextId: 3,
-  byId: new Map([
-    [0, { name: "Manifest Ledger", url: "/api", filter: "alberto" }],
-    [1, { name: "END Ledger", url: "/api-end", filter: "end-labs" }],
-    [
-      2,
-      { name: "Legacy Manifest Ledger", url: "/api/legacy", filter: "legacy" },
-    ], // Hidden from UI
-  ]),
-}
-
-interface NetworkActions {
-  getNetworks: () => NetworkParams[]
-  getActiveNetwork: () => NetworkParams | undefined
-  getLegacyNetworks: () => NetworkParams[]
-  createNetwork: (n: NetworkParams) => void
+interface NetworksActions {
   setActiveId: (id: NetworkId) => void
-  updateNetwork: (id: NetworkId, n: NetworkParams) => void
+  getActiveNetwork: () => NetworkInfo
+  getLegacyNetworks: () => NetworkInfo[]
+  getNetworks: () => Map<NetworkId, NetworkInfo>
+  createNetwork: (n: NetworkInfo) => void
+  updateNetwork: (id: NetworkId, n: NetworkInfo) => void
   deleteNetwork: (id: NetworkId) => void
```

Comment:
Thanks for cleaning up NetworkStore. [Zustand](https://docs.pmnd.rs/zustand/getting-started/introduction) pretends to be all "light weight" but you can actually get pretty gnarly pretty fast. I think we should've treated it more like a closed module and been more careful with the interface or even used something like `useReducer` so an action in one part of the app (changing a Network) could impact the state of a different part (the current identity). It's probably not too late, but it would be a substantial refactor.

Reply:
The separation between Context and Store is a little weird at the moment.